op-node/rollup/derive/span_channel_out.go

```diff
@@ -166,20 +194,24 @@ func (co *SpanChannelOut) AddSingularBatch(batch *SingularBatch, seqNum uint64)
 	if err = co.compress(); err != nil {
 		return err
 	}
-	co.lastCompressedRLPSize = co.activeRLP().Len()
```

Comment:
I see this is moved into the call to `compress`. I think that's probably a better place for it. Trying to remember why i had it up here in the first place.

Reply:
yeah it felt better to move the modification of the state variable into the method that owns the modification. Did the same for the `sealerRLPBytes`, setting it in `ensureOpenSpanBatch`.